anaconda-modules.ks.in

```diff
@@ -0,0 +1,48 @@
+#version=DEVEL
+#test name: anaconda-modules
+
+# Verify that only the enabled Anaconda DBus modules are started.
+
+# Use defaults.
+%ksappend repos/default.ks
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+%post --nochroot
+
+# Get the activated Anaconda DBus modules.
+less /tmp/syslog | grep "Activating service name" | \
+grep "org.fedoraproject.Anaconda.Modules" | cut -d"'" -f2 \
+| sort > /tmp/generated.out
+
+# Generate the expected output.
+cat > /tmp/expected.out <<EOF
```

Comment:
Please add space between `<<` and `EOF`.

Reply:
Fixed.